json/embeddings/embeddings-docs.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema#",
+  "$id": "https://impresso.github.io/impresso-schemas/json/embeddings/embeddings-docs.schema.json",
+  "title": "Document Embeddings JSON Schema",
+  "description": "A representation for the vector embedding representation of content items.",
+  "type": "object",
+  "properties": {
+    "id": {
```

Comment:
Perhaps to be revised according to the decision taken in https://github.com/impresso/impresso-schemas/issues/40